src/core/default_appearance.js

```diff
@@ -321,6 +321,22 @@ endcmap CMapName currentdict /CMap defineresource pop end end`;
       `/${this.fontName.name} ${numberToString(newFontSize)} Tf`,
     ];
 
+    const { resources } = this;
+    strokeAlpha =
+      typeof strokeAlpha === "number"
+        ? Math.min(1, Math.max(0, strokeAlpha))
+        : 1;
```

Comment:
Is this sort of clamping what e.g. Adobe Reader does, for a value outside of the [0, 1] range?

Please note that I've not checked, but wanted to ask if we should perhaps just do something like the following instead:
```suggestion
    strokeAlpha =
      typeof strokeAlpha === "number" && strokeAlpha >= 0 && strokeAlpha <= 1
        ? strokeAlpha
        : 1;
```